### ISSUE TITLE
IJulia not showing output from package manager mode

### ISSUE THREAD

**Issue by bkamins** [JuliaLang/IJulia.jl#1017]
Users of my notebooks are reporting to me that they stopped seeing output of package manager in Jupyter Notebook.

I have just checked and indeed I can reproduce it. Here is an example (both cells 1 and 2 were just executed):
![image](https://user-images.githubusercontent.com/6187170/127279463-03b1d3a0-7c90-4f0b-83da-4dd92071208e.png)

Environment configuration:
```
julia> versioninfo()
Julia Version 1.6.1
Commit 6aaedecc44 (2021-04-23 05:59 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-10850H CPU @ 2.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake)

julia> Pkg.status("IJulia")
      Status `C:\Users\bogum\.julia\environments\v1.6\Project.toml`
  [7073ff75] IJulia v1.23.2
```

CC @KristofferC

**Comment by KristofferC** [comment]
> Julia Version 1.6.1

Should be fixed in 1.6.2

**Comment by bkamins** [comment]
Indeed. Thank you!
